gsoc/admin.py

```diff
@@ -1,4 +1,5 @@
 from .models import *
+from .filters import GSoCYearFilter
```

Comment:
dont need to make a whole new file and class for this

Reply:
Fixed